### ISSUE TITLE
making nice images with Phoenix

### ISSUE THREAD

**Issue by sponce** [HSF/phoenix#451]
One thing people from the outreach like to get is nice pictures which can be used then by media in general and newspapers in particular. These picture need decent resolution (4K or more typically). Now phoenix is only able to produce screenshots, so requires you have a very big screen if you want a nice picture.

Is there anyway to get a "screenshot" of a virtual screen of a size different from the actual one ? Alternatively does threejs support svg as an output for images ?

**Comment by gauravsharma21** [comment]
@sponce @EdwardMoyse Can I work on this feature? 
I was thinking about adding another option inside Import and Export options within the Phoenix Iconbar labeled as "Export SVG"

**Comment by sponce** [comment]
You're more than welcome !

**Comment by Somya-Bansal159** [comment]
I have tried to add an option for exporting SVG. I am successfully able to download an SVG image. But, there are a few issues:
1. The info panel is not exported. I haven't found a way to export HTML as SVG.
2. SVG file has all the objects including the occluded ones. So, its size is big.

**Comment by JanEricNitschke** [comment]
I would like to reiterate the desire for this feature (and/or also https://github.com/HSF/phoenix/issues/581).

I tried to emulate a device with a higher resolution and take a better screenshot with that but sadly in the end they look the same still.

**Comment by EdwardMoyse** [comment]
I think quite a lot of this has been added. Did you give it a try - you can now make very large screenshots.

**Comment by JanEricNitschke** [comment]
Yeah, i did, both with https://phoenixatlas.web.cern.ch/PhoenixATLAS/ as well as cloned and run locally.

I can do the preset 3860 by 2610 but that still looks quite rough to me when you zoom in even a tiny bot. Setting it bigger to twice each just gives me a grey picture.

![screencapture00](https://github.com/HSF/phoenix/assets/47750513/16cca174-7c5c-4710-bacb-9528428fd112)
![screencapture](https://github.com/HSF/phoenix/assets/47750513/791133e5-0ae7-42bd-b9eb-a18d8f819ff4)

**Comment by EdwardMoyse** [comment]
Which browser? If you can, try with another as I believe they have different limits.

**Comment by sponce** [comment]
I think this one is just a duplicate of #581 and we only need to find the time to implement the second point described there, and implement a solution using setViewOffset and recomposition of the image
